### ISSUE TITLE
Don't pin releases of requirements

### ISSUE THREAD

**Issue by fabaff** [barnumbirr/coinmarketcap#40]
For Home Assistant there is an integration which uses `coinmarketcap`. In the past we were running into issues if a dependency was pinned to a specific release of one of their requirements. 

`coinmarketcap` is pinning `requests==2.18.4` and `requests_cache==0.4.13`. Could you please consider to ease this?

Thanks

**Comment by barnumbirr** [comment]
Hello,

I sure could ease the pinned requirements, what would be useful in your case?

Cheers.

**Comment by fabaff** [comment]
Minimum version would work or if you are planning to maintain it no version at all.

**Comment by barnumbirr** [comment]
So something like this would work for you:

```python
install_requires=[
    'requests>=2.18.4',
    'requests_cache>=0.4.13'
]
```

**Comment by fabaff** [comment]
Yes, because we try to stay up-to-date with our core requirements and requests is one of them.

**Comment by barnumbirr** [comment]
Ok, I don't think this should have a big impact.

There's just something I don't understand (if you don't mind clearing that up for me): 

Lastest version for ```requests``` is ```2.18.4```, so the version I have pinned in the setup file and don't see ```home-assistant``` requiring ```requests-cache``` as a dependency. I don't mind modifing this if it makes your life easier but I just don't seem to grasp where the initial issue is. (Sorry for being daft on this one)

Cheers.

**Comment by barnumbirr** [comment]
v.5.0.3 is now on Pypi and includes fixes in 4e90337bdb467010d310bd796b898acef67d370a.

**Comment by fabaff** [comment]
Thanks
